todo.css

```diff
@@ -3,23 +3,23 @@
   box-shadow: 0 1px 3px 0 rgba(0,0,0,.1);
 }
 
- .container {
+.container {
   width:700px;
-  margin-right: auto;
-  margin-left : auto;
+  margin: 0 auto;
 }
 
 .box1 {
-    padding: 0.5em 1em;
-    margin: 2em 0;
-    width: 80%;
-    color: #5d627b;
-    background: white;
-    box-shadow: 0 3px 5px rgba(0, 0, 0, 0.22);
+  padding: 0.5em 1em;
+  margin: 2em 0;
+  width: 80%;
+  color: #5d627b;
+  background: white;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.22);
 }
+
 .box1 p {
```

Comment:
I personally don't like to use the element here.
I'd create a `.box1 .another-class` instead of the element directly